x-pack/plugins/alerting/server/rules_client/lib/retry_if_bulk_edit_conflicts.ts

```diff
@@ -76,12 +80,17 @@ export const retryIfBulkEditConflicts = async (
     const results = [...accResults, ...resultSavedObjects.filter((res) => res.error === undefined)];
     const apiKeysToInvalidate = [...accApiKeysToInvalidate, ...localApiKeysToInvalidate];
     const errors = [...accErrors, ...localErrors];
+    // Create array of unique skipped rules by id
+    const skipped = [
+      ...new Map([...accSkipped, ...localSkipped].map((item) => [item.id, item])).values(),
```

Comment:
Not sure I understand what you mean.

The Map is necessary because we need to de-duplicate `skipped` rules which might have the same `id`, which might have been passed recursively to the function in the case of conflicts while updating (part of the OCC logic)